src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -452,6 +450,9 @@
   <ItemGroup>
     <EmbeddedResource Include="$(_ILLinkRuntimeRootDescriptorFilePath)" />
   </ItemGroup>
+  <ItemGroup>
+    <Service Include="{508349B6-6B84-4DF5-91F0-309BEEBAD82D}" />
```

Comment:
Was this intentional?

Reply:
No, will revert.  VS added this automatically.